README.md

```diff
@@ -2,537 +2,25 @@
-          // Appium v2 does not support relative paths.
-          app: `${__dirname}/wikipedia.zip`,
-          bundleId: 'org.wikimedia.wikipedia',
-          newCommandTimeout: 0
-        }
-      }
-    },
-
-    'app.ios.real': {
-      extends: 'app',
-      'desiredCapabilities': {
-        // More capabilities can be found at https://github.com/appium/appium-xcuitest-driver#capabilities
-        browserName: null,
-        platformName: 'ios',
-        // `appium:options` is not natively supported in Appium v1, but works with Nightwatch.
-        // If copying these capabilities elsewhere while using Appium v1, make sure to remove `appium:options`
-        // and add `appium:` prefix to each one of its capabilities, e.g. change 'app' to 'appium:app'.
-        'appium:options': {
-          automationName: 'XCUITest',
-          // While Appium v1 supports relative paths, it's more safe to use absolute paths instead.
-          // Appium v2 does not support relative paths.
-          app: `${__dirname}/wikipedia.zip`,
-          bundleId: 'org.wikimedia.wikipedia',
-          newCommandTimeout: 0,
-          // add udid of the device to run tests on. Or, pass the id to `--deviceId` flag when running tests.
-          // device id could be retrieved from Xcode > Window > "Devices and Simulators" window.
-          // udid: '00008030-00024C2C3453402E'
-        }
-      }
-    },
-   }
-   ```
-5. Add the following sample test to your test-suite (`test/wikipedia-ios.js`):
-   ```js
-   describe('Wikipedia iOS app test', function() {
-     before(function(app) {
-       app.click('xpath', '//XCUIElementTypeButton[@name="Skip"]');
-     });
-
-     it('Search for BrowserStack', async function(app) {
-       app
-         .useXpath()
-         .click('//XCUIElementTypeSearchField[@name="Search Wikipedia"]')
-         .sendKeys('//XCUIElementTypeSearchField[@name="Search Wikipedia"]', 'browserstack')
-         .click('//XCUIElementTypeStaticText[@name="BrowserStack"]')
-         .waitUntil(async function() {
-           // wait for webview context to be available
-           const contexts = await this.appium.getContexts();
-
-           return contexts.length > 1;
-         }, 5000)
-         .perform(async function() {
-           // switch to webview context
-           const contexts = await this.appium.getContexts();
+Nightwatch helper-tool can be used on both Android and IOS devices. You can use the guides below to use it on your devices.
 
-           await this.appium.setContext(contexts[1]);
-         })
-         .useCss()
-         .assert.textEquals('.pcs-edit-section-title', 'BrowserStack');  // command run in webview context
-     });
-   });
-   ```
-6. (**Real Device**) Run the following command to get the *UDID*:
-   ```sh
-    system_profiler SPUSBDataType | sed -n '/iPhone/,/Serial/p' | grep 'Serial Number:' | awk -F ': ' '{print $2}'
-    ```
+- [Android](docs/ANDROID.md)
+- [IOS](docs/IOS.md)
```

Comment:
Keep the format for this to be the same as before. Also, we should bifurcate the usage docs based on Mobile web testing and Mobile app testing, and not Android and iOS. Sorry if I mentioned dividing the docs for Android and iOS before.

Reply:
Ok will bifurcate into web and android testing